bahmnicore-api/src/test/java/org/bahmni/module/bahmnicore/contract/patient/mapper/PatientResponseMapperTest.java

```diff
@@ -54,7 +55,7 @@ public void setUp() throws Exception {
         visits.add(visit);
         PowerMockito.when(visitService.getActiveVisitsByPatient(patient)).thenReturn(visits);
         PowerMockito.when(Context.getVisitService()).thenReturn(visitService);
-        PowerMockito.when(bahmniVisitLocationService.getVisitLocation(any(String.class))).thenReturn(location);
+        PowerMockito.when(bahmniVisitLocationService.getVisitLocation(eq(null))).thenReturn(location);
```

Comment:
@angshu with the latest mockito version if the parameter doesn't exactly match with the value that is being passed as part of running test case then the value it's expected to return is not returning. This is the reason why we see lot of places we have replace with eq(null) because that the value that has passed as apart of test run.